elementor/elementor
# Tweak: AI Context Promotions [ED-13779]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary
AI Context Promotions

## Description
Adds promotions to setup the AI context in all AI entry points (Text with AI, AI containers, etc)

## Test instructions
Open each of the AI feature and expect to see a one time promotion, upon dismissing the promotion it should not re-appear for that AI feature

1. Text with AI
2. Image with AI
3. Code with AI
4. Variations with AI

## Quality assurance
